src/dispatch/incident/messaging.py

```diff
@@ -307,6 +309,7 @@ def send_incident_created_notifications(incident: Incident, db_session: SessionL
         "severity_description": incident.incident_severity.description,
         "priority": incident.incident_priority.name,
         "priority_description": incident.incident_priority.description,
+        "visibility": incident.visibility,
```

Comment:
@jneelamekam should we bump it further up in the message to ensure people read the description? Perhaps between description and status?

Reply:
Updated the position of where we display changes in the recent commit